### ISSUE TITLE
webutil.useradd marked as deprecated:: 2016.3.0 by mistake?

### ISSUE THREAD

**Issue by boltronics** [saltstack/salt#42870]
### Description of Issue/Question
I'm unclear as to why `webutil.useradd` was marked as deprecated. What's the replacement? Why aren't the other functions in `webutil` deprecated? Smells like the notice is a mistake.

I'm also confused as to why we have `htpasswd` implement `webutil` as virtual, but it's not listed at the top of the docs [here](https://docs.saltstack.com/en/latest/ref/modules/all/index.html). As for the `htpasswd` state module, it's even less clear that this is implemented as a virtual, and one needs to use `webutil.user_exists` instead of `htpasswd.user_exists`. Very easy to overlook this in the one example when it's not pointed out.

### Setup
Stock 2017.7.0 with the `htpasswd` command installed in Salt's path.

### Steps to Reproduce Issue
```
$ sudo salt-call sys.doc webutil.useradd
local:
    ----------
    webutil.useradd:
        
            Add a user to htpasswd file using the htpasswd command. If the htpasswd
            file does not exist, it will be created.
        
            .. deprecated:: 2016.3.0
        
```

### Versions Report
```
$ sudo salt-call --versions-report
Salt Version:
           Salt: 2017.7.0
 
Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.2
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.3
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: Not Installed
      pycparser: 2.10
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.4.1
            ZMQ: 4.0.5
 
System Versions:
           dist: debian 8.9 
         locale: UTF-8
        machine: x86_64
        release: 3.16.0-4-amd64
         system: Linux
        version: debian 8.9 
 
```

**Comment by garethgreenaway** [comment]
@rallytime Looks like you updated the deprecation version on the webutil module.  Care to comment on the deprecation?

**Comment by rallytime** [comment]
@boltronics @garethgreenaway Looks like something just got crossed here in the docs. The function that was deprecated was `useradd_all` in _favor_ of `useradd` (see #29689). The `deprecated` doc section must have some how ended up in the docs for `useradd` from a merge, because in #38421, that doc line is still in the `useradd_all` section.

I'll just remove that `deprecated` mention from the docs. Thanks for letting us know @boltronics!

**Comment by rallytime** [comment]
Fixed in #42887
